docs/README.md

```diff
@@ -425,6 +426,7 @@ In order to apply CSS customization in this and other blocks, follow the instruc
 | `searchNotFoundWhatToDoDots`          |
 | `searchNotFound`                      |
 | `searchResultContainer`               |
+| `seeMoreButton`                       |
```

Comment:
Could you add it below `resultGallery` handle?

Reply:
@guerreirobeatriz don't we have to maintain alphabetical order?